### PR TITLE
Mobile - Font 1px bigger + misc tweaks

### PR DESCRIPTION
**Pull request by cecileboucheron**

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits ce29c60281a3...bd95ed1a2263 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495475455123-ce29c60281a3-bd95ed1a2263-linux/Folders_Files-You_can_unlock_Public-ce29c60281a3.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495475455123-ce29c60281a3-bd95ed1a2263-linux/Folders_Files-You_can_unlock_Public-bd95ed1a2263.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495475455123-ce29c60281a3-bd95ed1a2263-linux/Folders_Files-You_can_unlock_Private-ce29c60281a3.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495475455123-ce29c60281a3-bd95ed1a2263-linux/Folders_Files-You_can_unlock_Private-bd95ed1a2263.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 038bbb4c54e7...0a54fe299afe introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495482700204-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Public-038bbb4c54e7.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495482700204-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Public-0a54fe299afe.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495482700204-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Private-038bbb4c54e7.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495482700204-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Private-0a54fe299afe.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 038bbb4c54e7...0a54fe299afe introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495487422249-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Public-038bbb4c54e7.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495487422249-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Public-0a54fe299afe.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495487422249-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Private-038bbb4c54e7.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495487422249-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Private-0a54fe299afe.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 038bbb4c54e7...0a54fe299afe introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495491768465-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Public-038bbb4c54e7.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495491768465-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Public-0a54fe299afe.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495491768465-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Private-038bbb4c54e7.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495491768465-038bbb4c54e7-0a54fe299afe-linux/Folders_Files-You_can_unlock_Private-0a54fe299afe.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits a593148b2d2a...8f388e3e6958 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495494710804-a593148b2d2a-8f388e3e6958-linux/Folders_Files-You_can_unlock_Public-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495494710804-a593148b2d2a-8f388e3e6958-linux/Folders_Files-You_can_unlock_Public-8f388e3e6958.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495494710804-a593148b2d2a-8f388e3e6958-linux/Folders_Files-You_can_unlock_Private-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495494710804-a593148b2d2a-8f388e3e6958-linux/Folders_Files-You_can_unlock_Private-8f388e3e6958.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

File: shared/local-debug.native.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ let config: {[key: string]: any} = {
 
 if (__DEV__ && true) {
   config.isDevApplePushToken = true
-  config.clickableVisible = true
+  config.clickableVisible = false
```
Comment (chromakode) [inline shared/local-debug.native.js]: Personally, :+1: to doing this, but we should discuss it and PR it separately. Please take it out of this changeset.

**Comment by keybase-ci-visdiff** [comment]
The commits a593148b2d2a...56975645c66e introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499203388-a593148b2d2a-56975645c66e-linux/Folders_Files-You_can_unlock_Public-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499203388-a593148b2d2a-56975645c66e-linux/Folders_Files-You_can_unlock_Public-56975645c66e.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499203388-a593148b2d2a-56975645c66e-linux/Folders_Files-You_can_unlock_Private-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499203388-a593148b2d2a-56975645c66e-linux/Folders_Files-You_can_unlock_Private-56975645c66e.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits a593148b2d2a...6ce91327b511 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499766590-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Public-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499766590-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Public-6ce91327b511.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499766590-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Private-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495499766590-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Private-6ce91327b511.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits a593148b2d2a...6ce91327b511 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Folders_Files-You_can_unlock_Public** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495501523600-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Public-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495501523600-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Public-6ce91327b511.png)  
   Folders_Files-You_can_unlock_Public comparison
 * Changed: **Folders_Files-You_can_unlock_Private** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495501523600-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Private-a593148b2d2a.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495501523600-a593148b2d2a-6ce91327b511-linux/Folders_Files-You_can_unlock_Private-6ce91327b511.png)  
   Folders_Files-You_can_unlock_Private comparison
</summary>

**Pull request merged**